subsys/net/ip/net_context.c

```diff
@@ -1175,6 +1272,12 @@ int net_context_send(struct net_pkt *pkt,
 
 	k_mutex_lock(&context->lock, K_FOREVER);
 
+	if (IS_ENABLED(CONFIG_NET_SOCKETS_PACKET) &&
```

Comment:
Please add a comment that only sendto() is supported for AF_PACKET sockets.

Reply:
ok